relation_engine_server/README.md

```diff
@@ -460,9 +460,9 @@ make test
 
 ## Deployment
 
-The docker image is pushed to Docker Hub when new commits are made to master. The script that runs when pushing to docker hub is found in `hooks/build`.
+Deploy the image by running the `scripts/docker_deploy` script.
```

Comment:
This should be clarified -- something like "The image can be manually deployed to Dockerhub by running the `scripts/docker_deploy` script. Please note that only those with the appropriate credentials can push images to Dockerhub."

Reply:
Added